internal/backend/s3/s3.go

```diff
@@ -164,7 +164,7 @@ func isAccessDenied(err error) bool {
 	debug.Log("isAccessDenied(%T, %#v)", err, err)
 
 	var e minio.ErrorResponse
-	return errors.As(err, &e) && e.Code == "Access Denied"
+	return errors.As(err, &e) && e.Code == "AccessDenied"
```

Comment:
LGTM. Sorry for screwing this up.

Reply:
Not a problem.
I'm glad to give some contribution.